pkgs/swift2objc/lib/src/parser/_core/utils.dart

```diff
@@ -38,10 +40,16 @@ extension TopLevelOnly<T extends Declaration> on List<T> {
       ).toList();
 }
 
+/// If fragment['kind'] == kind, returns fragment['spelling']. Otherwise returns
```

Comment:
nit: Wrap code with backticks. `[]` might do some linking stuff in markdown otherwise.

Reply:
Done